turtle_tf2_cpp/src/dynamic_frame_tf2_broadcaster.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <geometry_msgs/msg/transform_stamped.hpp>
+
+#include <rclcpp/rclcpp.hpp>
+#include <tf2_ros/transform_broadcaster.h>
+
+#include <memory>
+
+using namespace std::chrono_literals;
+
+class DynamicFrameBroadcaster : public rclcpp::Node
+{
+public:
+  DynamicFrameBroadcaster()
+  : Node("dynamic_frame_tf2_broadcaster")
+  {
+    tf_publisher_ = std::make_shared<tf2_ros::TransformBroadcaster>(this);
+    timer_ = this->create_wall_timer(
+      100ms, std::bind(&DynamicFrameBroadcaster::broadcast_timer_callback, this));
+  }
+
+private:
+  void broadcast_timer_callback()
+  {
+    geometry_msgs::msg::TransformStamped t;
+
+    rclcpp::Time now = this->now();
+    double pi = 2 * acos(0.0);
```

Comment:
A couple things:
* I rather use a constant, like in https://www.codespeedy.com/pi-constant-in-cpp/
* Doing it this way, it could be a `static const` in the class or loop, or a constant outside of the class

Reply:
I updated it in https://github.com/ros/geometry_tutorials/pull/52/commits/4cc74eb54e8ae97262dc612a2c6b18d63bd42c2f. Is it better that way?